apps/pkcs12.c

```diff
@@ -596,7 +599,8 @@ int pkcs12_main(int argc, char **argv)
                             key_pbe, cert_pbe, iter, -1, keytype);
 
         if (p12 == NULL) {
-            ERR_print_errors(bio_err);
+            BIO_printf(bio_err, "Error creating PKCS12 structure for %s\n",
+                       outfile);
```

Comment:
If you look anywhere else, you'll see that we do `ERR_print_errors()` *in addition* to whatever the app wants to print.  The app error is not a replacement, so while I see nothing wrong with the `BIO_printf()` call, I do think it's wrong to remove the `ERR_print_errors()` call.  The latter is likely to give additional details.

Reply:
That's understood.
Note that I did not remove `ERR_print_errors(bio_err);` but moved it after the `export_end:` label within `pkcs12_main()`
because outputting the error queue may be helpful also other (error) cases that end up there.